resources/lang/jp/auth.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+return [
+    'failed'                   => '記録された情報に一致していません。',
+    'throttle'                 => '多数のログイン、:seconds 秒後に再度ログインしてください。',
+    'password'                 => 'パスワード',
+    'createaccount'            => 'アカウントの新規登録',
+    'forgotpassword'           => 'パスワード忘れ方はこちら',
+    'fullname'                 => '氏名(フルネーム)',
+    'emailaddress'             => 'メールアドレス',
+    'fillcaptcha'              => 'Fill out the captcha',
+    'tocaccept'                => '登録することで, あなたは規約に同意する。',
+    'register'                 => '新規登録',
+    'registrationpending'      => '登録待機中',
+    'pendingmessage'           => 'ANAバーチャルグループへの新規登録誠にありがとうございます。私たちの管理員がアカウントの確認をしています。少々お待ちください！(1300z ~ 2200zは管理員の休憩時間です。多少の遅れが生じます)',
+    'registrationsubmitted'    => '登録提出済み',
+    'registrationconfirmation' => '登録の確認',
+    'confirmationmessage'      => 'ANAバーチャルグループへの新規登録誠にありがとうございます。私たちの管理員がアカウントの確認をしています。少々お待ちください！(1300z ~ 2200zは管理員の休憩時間です。多少の遅れが生じます)',
```

Comment:
This is in reference to a specific VA. Please change this to something more generic, in line with the original